src/components/Notifications/TimeConfig.tsx

```diff
@@ -45,26 +60,27 @@ export const TimeConfigComponent: React.FunctionComponent = () => {
         setShowCustomSelect(true);
     }, []);
 
-    const handleTimeSelect = React.useCallback(() => {
+    const handleTimePrefSelect = React.useCallback((time) => {
+        setTimeSelect(time);
         setIsOpen(false);
-    }, []);
+    }, [ ]);
+
+    const handleButtonSave = React.useCallback(() => {
+        if (timeSelect) {
+            const mutate = saveTimePreference.mutate;
+            mutate({
+                body: timeSelect
+            }).then((response) => {
+                if (response.status === 200) {
```

Comment:
Lets change this to 204 - the other PR has been merged
```suggestion
                if (response.status === 204) {
```